src/Pages/UserManager/EditUserInfo.js

```diff
@@ -263,7 +279,7 @@ export default function EditUserInfo(props) {
                         checked={!!emailVerified}
                         onChange={(e) => {
                           setDataWasChanged(true);
-                          setEmailVerified(e.target.checked);
+                          setEmailVerified(newEmailVerified);
```

Comment:
`newEmailVerified` is not defined anywhere

Reply:
done